### DISCUSSION TITLE
Here it is ;-) Hardware is setup

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#269] [Show and tell]
As I do not have skills to work with wood , I did it with V-Slots 2020 and plastic panels

![IMG_7623](https://github.com/photobooth-app/photobooth-app/assets/37927465/f1894904-c6f8-4ef5-9006-d5a0b9f45a99)

**Comment by mgineer85** [comment]
Wow, thats massive :)   Looks great
